buildtools/src/main/resources/pulsar/checkstyle.xml

```diff
@@ -39,6 +39,9 @@ page at http://checkstyle.sourceforge.net/config.html -->
         <property name="max" value="120"/>
         <property name="severity" value="error"/>
 
+        <!-- Checks only for Java files. -->
+        <property name="fileExtensions" value="java"/>
```

Comment:
this should be a separate PR

Reply:
OK. I include this change in the patch because after I move config files under the module, this rule takes effect and reports failure. I can do it in a separate PR though.

Reply:
See https://github.com/apache/pulsar/pull/17168.